lib/get-package-versions.js

```diff
@@ -0,0 +1,56 @@
+const fs = Npm.require('fs');
+
+// Based on https://github.com/meteor/meteor/blob/d9db4f52f2ea6d706a25156768ea42e1fbb8f599/tools/utils/utils.js#L250
+function parsePackageAndVersion (packageAtVersionString) {
+  const separatorPos = Math.max(packageAtVersionString.lastIndexOf(' '),
+    packageAtVersionString.lastIndexOf('@'));
+
+  if (separatorPos < 0) {
+    return;
+  }
+
+  const packageName = packageAtVersionString.slice(0, separatorPos);
+  const version = packageAtVersionString.slice(separatorPos + 1);
+
+  return { package: packageName, version };
+}
+
+// Based on https://github.com/zodern/meteor-package-versions/blob/master/plugin.js
+export function getPackageVersions () {
```

Comment:
This needs to run in the compilation phase, wouldn't work in the production bundle otherwise. Need to revert it when the version issue is resolved or we have a proper solution.